docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -17,6 +17,102 @@ import TimelineDot from '@mui/lab/TimelineDot';
 import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
 
 const entries = [
+  {
+    title: 'MUI X v8',
+    description: 'Cutting-edge features, enhanced customization, and full support for native ESM.',
+    date: new Date(2025, 3, 17),
+    url: 'https://mui.com/blog/mui-x-v8/',
+    highlightList: [
+      {
+        title: 'Data Grid – Pivoting [Premium]',
```

Comment:
WDYT about using the regular Pro and Premium plan icons instead? 🤔

Reply:
didnt want to increase the scope of the component reading this to render the link. But it might have been easy.

Reply:
https://github.com/mui/mui-x/issues/17400